src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/DefendantResponseApplicantForCuiNotificationHandler.java

```diff
@@ -60,17 +60,26 @@ public List<CaseEvent> handledEvents() {
 
```

Comment:
This File has changes for 'Respond To Claim' journey notification event update for LiP Claimant.